### ISSUE TITLE
can I use GONE for ddRAD data without reference genome?

### ISSUE THREAD

**Issue by YuanshuPu** [esrud/GONE#42]
Dear Armando and fellow users,

I come across GONE and it's very interesting for me because I'm specifically interested in the most recent Ne of my populations (<100 generations). However, I have ddRADseq data without a reference genome (50k loci in 250bp length, with 4 SNPs per locus on average). I'm wondering, if it makes sense at all, to run GONE with a subset of 200 loci with the most SNPS (hence, 23-50 SNPs per locus), since at maximum 200 chromosomes are allowed in the software. Otherwise, do you perhaps have any suggestions for calculating recent Ne with my type of data?

Best wishes and many thanks in advance,
Yuanshu

**Comment by armando-caballero** [comment]
No. You cannot get estimates with as few as 23-50 SNPs per "chromosome" at all.
If you were able to ascribe SNPs to real chromosomes or large linkage fragments, yes.
You need at least hundreds of SNPs per chromosome for analysis.
Best wishes,
Armando.
